students/rachel_s/session03/list_lab.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+
+# Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+og_fruits = fruits.copy()
+
+# Display the list (plain old print() is fine…).
+print(fruits)
+
+# Ask the user for another fruit and add it to the end of the list.
+response = input("Please tell me another fruit > ")
+fruits.append(response.capitalize())
+
+# Display the list.
+print(fruits)
+
+# Ask the user for a number and display the number back to the user and the 
+# fruit corresponding to that number (on a 1-is-first basis). Remember that 
+# Python uses zero-based indexing, so you will need to correct.
+my_num = input("Pick a number between 1 and 6 > ")
+
+if int(my_num) in range(7):
```

Comment:
Minor: If user enters 0, this will pass, causing line 24 to crash (index out of bounds). I think you'd want to change the `range()` function params to avoid that.